src/lib/replication_ops.rs

```diff
@@ -148,15 +148,6 @@ pub fn replicate_request(
                     replication_sender,
                     get_replicate_message(db_name.to_string(), key.clone(), value.clone(), version),
                 );
-                replicate_web(
```

Comment:
Removing this since the consensus ops deal with the replication of the conflict key